plugin.video.hak5/addon.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.hak5"
+       name="Hak5"
+       version="1.0.0"
+       provider-name="Skipmode A1">
+  <requires>
+    <import addon="xbmc.python"                 version="2.25.0"/>
+	<import addon="script.module.beautifulsoup" version="3.0.8"/>
+	<import addon="script.module.requests"      version="2.4.3"/>
+    <import addon="plugin.video.youtube"        version="5.1.7"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+	<summary lang="en">Watch videos from hak5.com</summary>
```

Comment:
I'll alter it.

The language-tag should be this also?
    <language>en_gb</language>